frontend/packages/core/src/Network/errors.ts

```diff
@@ -0,0 +1,290 @@
+import type { AxiosError } from "axios";
```

Comment:
From reading the spec I now know these error details are already defined in [googleapis protos](https://github.com/googleapis/googleapis/blob/master/google/rpc/error_details.proto), are we unable to reuse them on the frontend? If so why cant we?

Reply:
there is a short bit in the spec about these not being available via a protojs package that's currently published :)

Reply:
Ah ok I i must of missed that part :) Thank you.